lib/helpers.coffee

```diff
@@ -32,9 +32,8 @@ module.exports = Helpers =
           resolve(data.stderr.join(''))
       if isNodeExecutable
         options.env ?= {}
-        for prop in process.env
-          if process.env.hasOwnProperty(prop) and prop isnt 'OS'
-            options.env[prop] = process.env[prop]
+        for prop,value of process.env
```

Comment:
You could have simply changed the `for ... in` to be `for ... of` to fix it. None of the changes you made were bad, it's just that you have a very small fix mixed in with enough reformatting that I couldn't find the actual fix.

Now that I see the functional change I don't doubt it fixed it.